samples/qa-scroller/src/index.js

```diff
@@ -2,14 +2,13 @@ import {createRoot} from 'react-dom/client';
 
```

Comment:
There is build error

Unable to generate prerender of app state HTML for en-US:
TypeError: Cannot read property 'undefined' of undefined

Reply:
`qa-voice-control` and `qa-scroller` needs more time to debug. But it's been errored with react17, so it seems not related to react18. Let's revisit this later.